homeassistant/components/knx/config_flow.py

```diff
@@ -338,14 +330,66 @@ async def async_step_secure_tunnel_manual(
             errors=errors,
         )
 
+    async def async_step_secure_routing_manual(
+        self, user_input: dict | None = None
+    ) -> FlowResult:
+        """Configure ip secure routing manually."""
+        errors: dict = {}
+
+        if user_input is not None:
+            try:
+                if len(bytes.fromhex(user_input[CONF_KNX_ROUTING_BACKBONE_KEY])) != 16:
+                    raise ValueError
```

Comment:
This should abort and show the error to the user.

Reply:
Oh wait you do that the line below? Why not just if-check and add to errors?

Reply:
I don't want to abort, but only show an error - so the user has the chance to fix it (eg. missed 1 char when copy&pasting).

`bytes.fromhex()` raises `ValueError` for non-hex strings (like "g") so I figured I use the same for wrong length.

Shall I use something more verbose like
```py
            try:
                 key_bytes = bytes.fromhex(user_input[CONF_KNX_ROUTING_BACKBONE_KEY])
            except ValueError:
                 errors[CONF_KNX_ROUTING_BACKBONE_KEY] = "invalid_backbone_key"
            if len(key_bytes) != 16:
                 errors[CONF_KNX_ROUTING_BACKBONE_KEY] = "invalid_backbone_key"  # maybe add a different string for invalid length
            if not errors:
```
?